packs/equipment/shield-augmentation.json

```
@@ -148,12 +148,13 @@
                 "rollOption": "shield-augmentation"
             },
             {
-                "definition": [
-                    "item:group:shield"
-                ],
-                "key": "AdjustStrike",
+                "itemType": "weapon",
+                "key": "ItemAlteration",
                 "mode": "add",
-                "property": "weapon-traits",
+                "predicate": [
+                    "item:group:shield"
+                ],                     
```

```suggestion
                ],
```
Looks like that whitespace is on all of these.